tests/test-prog.c

```diff
@@ -267,6 +281,9 @@ int main(int argc, char **argv)
     void *handle = NULL;
     int rv = 1;
     int loop_cnt;
+    // Sadly this is not known at compile time, see https://www.openwall.com/lists/musl/2013/03/29/13
+    int is_musl;
```

Comment:
Could you move the local variable `is_musl` to [here](https://github.com/kubo/injector/blob/23f3db17bcb03e43d845764824eced610c7aa97a/tests/test-prog.c#L156-L159)?
It is used only for linux.

Reply:
Done